packages/turf-dissolve/test.js

```diff
@@ -1,13 +1,18 @@
 var turfDissolve = require('./index');
 var test = require('tape');
 var polys = require('./test/polys.json');
-var expectedOutput = require('./test/outputPolys.json');
+var expectedOutput = require('./test/outputPolysByProperty.json');
+var expectedOutput2 = require('./test/outputPolysWithNoProperty.json');
```

Comment:
Instead of hardcoding your expected results, add a `fs.writeSync` function using an environment variable `REGEN=true` to output your expected results.

Here's an example of a test case using this:

https://github.com/Turfjs/turf/blob/master/packages/turf-buffer/test/test.js#L34-L38

```javascript
    buffered.features = buffered.features.concat(normalize(fixture).features);
    if (process.env.REGEN) {
      fs.writeFileSync(
          __dirname+'/fixtures/out/'+path.split('/')[path.split('/').length-1],
          JSON.stringify(buffered, null, 2));
    }
    var expected = JSON.parse(fs.readFileSync(
        __dirname+'/fixtures/out/'+path.split('/')[path.split('/').length-1]));
    t.deepEqual(expected, buffered);
  });
```